### DISCUSSION TITLE
OKD Direct Upgrade from 4.7 to 4.8

### DISCUSSION THREAD

**Discussion by Mohammed-Idrees** [okd-project/okd#940] [Q&A]
Hi Team,
I'm trying to upgrade OKD from 4.7(4.7.0-0.okd-2021-05-22-050008) to the 4.8 version. When I execute the command **oc adm upgrade** it only shows the few patch versions of the OKD 4.7. However, I have tried to OKD 4.8.0-0.okd-2021-10-10-030117(which is not listed on the available list of upgrades) it throws this upgrade is not available. 
Please let me know how to directly upgrade from okd4.7 to  4.8
![image](https://user-images.githubusercontent.com/36164488/138229560-33ca2b1f-94b4-41de-a628-95c54f4e8067.png)

**Comment by vrutkovs** [comment]
See "Upgrades from" section in https://amd64.origin.releases.ci.openshift.org/releasestream/4-stable/release/4.8.0-0.okd-2021-10-10-030117 for available upgrade edges. Other upgrade edges are not recorded in the graph (as they were not tested by CI) and thus not available in `oc adm upgrade`.

In order to upgrade to 4.8 you'd have to jump via:
* 4.7.0-0.okd-2021-06-13-090745
* 4.7.0-0.okd-2021-08-07-063045 
* 4.7.0-0.okd-2021-09-19-013247
and finally to 4.8.0-0.okd-2021-10-10-030117

**Reply by Mohammed-Idrees** [reply]
Thank you @vrutkovs.
I will verify it from my end .
